src/util.js

```diff
@@ -27,10 +27,15 @@ export function convertToQueryParams(o){
       case 'brandId':
       case 'categoryId':
       case 'facetBy':
-        let x = _.map(value, function(value){
-          return attr + '=' + value;
-        });
-        return x.join('&');
+        if (_.isArray(value)) {
+          let x = _.map(value, function(value){
```

Comment:
lodash is used in lot of places and its usage is not optimzied. Will apply this feedback while refactoring this api-client.